src/main/java/com/github/technus/tectech/thing/metaTileEntity/multi/GT_MetaTileEntity_EM_EyeOfHarmony.java

```diff
@@ -981,29 +980,43 @@ public GT_Multiblock_Tooltip_Builder createTooltip() {
                                 + "%"
                                 + GRAY
                                 + " per tier (additive). ")
+                .addInfo("  > Low tier stabilisation field generators have a power output penalty.")
+                .addInfo(
+                        "     The power output penalty for Crude Stabilisation Field Generator is " + RED
```

Comment:
"The power output penalty for using Crude Stabilisation Field Generators is" sounds a bit better